k2/python/k2/fsa_algo.py

```diff
@@ -421,3 +421,23 @@ def fix_aux_labels(src_aux_labels: torch.Tensor,
         setattr(out_fsa, name, value)
 
     return out_fsa
+
+
+def invert(fsa: Fsa) -> Fsa:
+    '''Invert an FST, swapping the labels in the FSA with the auxiliary labels.
+
+    Noted this version only accepts fsa.aux_labels as RaggedInt, if it's a tensor,
```

Comment:
I think it would be cleaner if it were to copy the FSA and invert_() if it's not RaggedInt aux_labels?
Eventually we'll want to support autograd; the version I'm making will support it.

Reply:
OK, Will add it (maybe after 2 hours when i get back home)

Reply:
Updated, thanks!